.github/workflows/ci.yaml

```diff
@@ -319,8 +319,8 @@ jobs:
         run: |
           chmod +x ./output/bin/buildtools
           chmod +x ./output/bin/embedded-cluster-original
-          ./output/bin/embedded-cluster-original version metadata > version-metadata.json
-          ./output/bin/embedded-cluster-original version list-images > expected.txt
+          sudo ./output/bin/embedded-cluster-original version metadata > version-metadata.json
+          sudo ./output/bin/embedded-cluster-original version list-images > expected.txt
```

Comment:
...why do 'version' commands require sudo again?

Reply:
If instantiating the provider requires sudo, that implies some bad things to me

Reply:
version itself doesnt but anything that requires us to materialize does because we have a new tmp dir if i remember correctly